changelog.md

```diff
@@ -0,0 +1,101 @@
+# Changelog
+
+## [0.9-beta] - 2019-03-19
+
+This changelog entry indicates notable changes since the alpha-2 development release of OC4IDS, it is not intended to be a complete list of changes.
+
+In addition to the specific changes to schema and codelists noted below:
+
+* Various refinements and clarifications were made to schema and codelist descriptions.
+* Guidance on mapping values from OCDS was moved from the schema to the IDS and OCDS mapping section of the documentation.
+* Documentation was expanded and restructured.
+
+### Packaging
+
+* Add [project package schema](http://standard.open-contracting.org/infrastructure/latest/en/reference/package/), OC4IDS data must be published as part of a project package.
+
+### Schema updates
+
+* `sector` - use projectSector open codelist
+* `contractingProcess` - add required `id` field
+* `linkedRelease` - make `id` a required field
+* `variations` - renamed to `modifications`
+* `location` - add required `id` field
+
+### New codelists
+
+* projectSector codelist - add codelist for project sector
+
+### Codelist updates
+
+* projectStatus codelist - replace 'construction' with 'implementation'
+* variationType codelist - rename codelist to modificationType
+* partyRole codelist - add OC4IDS codes mentioned in schema and mapping:
+ * funder
+ * administrativeEntity
+* partyRole codelist - add codes from OCDS partyRole codelist:
+ * buyer
+ * procuringEntity
+ * supplier
+ * tenderer
+* partyRole codelist - remove PPP specific codes:
+ * bidder
+ * qualifiedBidder
+ * preferredBidder
+ * privateParty
+ * leadBank
+ * lender
+ * equityInvestor
+ * consortiaMember
+ * interestedParty
+ * grantor
+ * disqualifiedBidder
+ * socialWitness
+ * otherWitness
+ * notary
+* documentType codelist - remove PPP specific codes:
```

Comment:
Hyphenate PPP-specific

Reply:
Fixed in https://github.com/open-contracting/infrastructure/pull/129/commits/79f023e6e64995432db88d5782d6f5c04280d769